files/build_templates/docker_image_ctl.j2

```diff
@@ -11,6 +11,7 @@ start() {
     else
 
         docker run -d {{docker_image_run_opt}} \
+            --log-opt max-size=2M --log-opt max-file=5 \
```

Comment:
I just want to set the maximal log size for each container to 10 MB. Then overall it will be ~100 MB on a switch.

Then I choose 2M * 5, so there are at least 8M log size in long running. If choosing 5M * 2, then the log size will fluctuate between 5M ~ 10M, which is not so good if you need more history.